src/components/Book.js

```diff
@@ -1,13 +1,24 @@
 /* eslint-disable react/prop-types */
```

Comment:
- [ ] Let's not disable linter rules. These are here to help us write clean code.

Reply:
I've just done it